kahuna/public/js/components/gu-date/gu-date.js

```diff
@@ -46,6 +45,7 @@ guDate.directive('guDate', [function () {
             const input = root.querySelector('input.gu-date__value--hidden');
             const container = root.querySelector('.gu-date__container');
 
+            const pikaFieldFormat = 'DD-MM-YYYY';
```

Comment:
Can I suggest using `DISPLAY_FORMAT` defined above on L10? The `DD-MM-YYYY` format can be a little ambiguous for our American friends - is 03/04/2021 03 April or 04 March?

`DISPLAY_FORMAT` is `DD MMM YYYY` which will render 03/04/2021 as 03 Mar 2021, which removes the ambiguity.

Reply:
yes, make sense